tests/server/api/notificationRoutes.spec.js

```diff
@@ -6,36 +6,45 @@ const { app } = require('../../../server/app');
 const request = supertest(app);
 const {
   db,
-  models: { NotificationList, Notification },
+  models: { Notification },
 } = require('../../../server/db/models/associations');
+const User = require('../../../server/db/models/User');
 
 describe('Notification Routes', () => {
   const notifications = [
     {
       amount: 29.99,
       category: 'Electronics',
       isCash: false,
-      notificationListId: 1,
     },
     {
       amount: 5.0,
       category: 'Food',
       isCash: false,
-      notificationListId: 1,
     },
   ];
+  const newUser = {
+    username: 'michelleO',
+    email: 'mobama@gmail.com',
+    password: 'password123',
+  };
   beforeAll(async () => {
     await db.sync({ force: true });
-    await NotificationList.create();
     await Notification.bulkCreate(notifications);
+    let userToken = (await request.post('/api/users').send(newUser)).body;
+    console.log(userToken);
   });
   afterAll(async () => {
     await db.close();
   });
 
   xtest('GET /api/notification returns all notifications', async (done) => {
-    const response = await request.get('/api/notification');
-    expect(response.body.length).toBe(2);
+    const response = await request.get('/api/notification', {
```

Comment:
I don't think this is going to work. The number of notifications in the db will change and the tests will always fail. You are better off writing your own dataset like we did before.